pyccel/parser/syntactic.py

```diff
@@ -741,11 +737,11 @@ def fill_types(ls):
 
         body = stmt.body
 
-        if 'sympy' in decorators.keys():
+        if 'sympy' in decorators:
             # TODO maybe we should run pylint here
             stmt.decorators.pop()
             func = SympyFunction(name, arguments, [],
-                    [stmt.__str__()])
+                    [str(stmt)])
```

Comment:
This should fit in a single line, which is more readable

Reply:
Fixed 772f82f8

Reply:
Not sure how I got round this before, probably because of the broken line. In any case I now have a coverage error but I don't know enough about what this is supposed to do to add a test

Reply:
See #692. There were tests but they are deactivated, see #330

Reply:
All right. Can we ask the CoverageChecker to ignore this line?

Reply:
Not a problem, the coverage check is not required anyway